src/infrastructure/logger.cpp

```diff
@@ -31,37 +31,43 @@ bool initialize(MPI_Comm comm)
     return false;
   }
 
-  // separate streams: warnings and errors (we); info and debug (id)
-  slic::LogStream* idStream;
-  slic::LogStream* weStream;
+  // Separate streams for different message levels
+  slic::LogStream* iStream;   // info
+  slic::LogStream* dStream;   // debug
+  slic::LogStream* weStream;  // warnings and errors
 
-  std::string fmt_id = "[<LEVEL>]: <MESSAGE>\n";
+  std::string fmt_i  = "<MESSAGE>\n";
+  std::string fmt_d  = "[<LEVEL>]: <MESSAGE>\n";
   std::string fmt_we = "[<LEVEL> (<FILE>:<LINE>)]\n<MESSAGE>\n\n";
 
   // Only create a parallel logger when there is more than one rank
   if (numRanks > 1) {
-    fmt_id = "[<RANK>]" + fmt_id;
+    fmt_i  = "[<RANK>] " + fmt_i;
```

Comment:
Why does this one have a space and the other two don't?

Reply:
So I can provide horrible inconsistency.. i'll fix it.

Reply:
Oh no it's right.  The other messages have prefix's for their messages.  fmt_i is the only one that doesn't. So the others have a previous space or newline.